TFT/src/User/API/Vfs/vfs.h

```diff
@@ -18,12 +18,12 @@ extern "C" {
 
 typedef enum
 {
-  TFT_SD,
-  TFT_USB_DISK,
-  BOARD_MEDIA,
-  BOARD_MEDIA_REMOTE,
-  REMOTE_HOST
-} FS_SOURCE;
+  FS_TFT_SD,
```

Comment:
my current PR already renamed those following the naming we provided in README.MD and config.ini files (we reported everywhere TFT SD, TFT USB, (REMOTE) ONBOARD MEDIA. Also FS_ is repetitive/ambiguous (FS_SOURCE was for File System source)

Reply:
Adding a common prefix to a set of enum makes them easier to find and use.  If `FS_` looks ambiguous then it can be replaced with `FS_SOURCE`. Although for these and other places `FS` is being used for the same meaning which makes it not ambiguous.
I am unable to understand how this makes the whole cleanup in your PR invalid? the only cleanup relating to these enums is renaming `TFT_USB_DISK` to `TFT_USB` and replacing `BOARD` with `ONBOARD`.

Reply:
hmm I preferred to avoid to use prefixes on the core files (Configuration.h, vfs, parseAck and interfaceCmd, Printing) because it is more compact and readable (of course we have to avoid to provide generic names such as RESUMED). When we use this def here and there it seems to me too much verbose to repeat a prefix. Also when we have variables already self explaining like infoFile.source it seems redundant to provide a prefix like FS_ when the rest of the def is not ambiguous (e.g. TFT_USB means any media on a TFT USB interface (a USB disk, pen drive, dongle etc.), ONBOARD_MEDIA means a media on a not known ONBOARD interface (could be SD, USB etc...). It seems to me also too much verbose HOST_STATUS_PRINTING. The prefix HOST_ was already clear and enough (e.g. HOST_PRINTING, HOST_PAUSED already indicate a status). Sure the type HOST_STATUS is correct